src/dotnet/UI.Blazor.App/Pages/Landing/DownloadButton.razor

```diff
@@ -1,8 +1,15 @@
-<button class="download-btn" data-href="@(Links.Apps.Android)">
-    <div class="text-04 text-headline-1">
-        Download for Android
-    </div>
-    <div>
-        <img draggable="false" src="/dist/images/landing/android.svg" alt="android">
+<button class="download-btn @Class">
```

Comment:
In all other cases button special classes starts with 'btn-' prefix, here it ('download-btn') finishes with '-btn' suffix.
Make it starts with standard suffix.